packages/web/src/index.ts

```diff
@@ -633,6 +515,14 @@ export const SplunkRum: SplunkOtelWebType = {
 	_experimental_getSessionId() {
 		return this.getSessionId()
 	},
+
+	_internalOnExternalSpanCreated() {
```

Comment:
this gets called from session recorder - when span is created we extend expiration (inactivity) time of session